src/main/java/com/group7/ecommerce/controller/api/ShipInfoController.java

```diff
@@ -0,0 +1,110 @@
+package com.group7.ecommerce.controller.api;
+
+import com.group7.ecommerce.dto.request.ShipInfoRequest;
+import com.group7.ecommerce.dto.response.ApiResponse;
+import com.group7.ecommerce.dto.response.ShipInfoResponse;
+import com.group7.ecommerce.service.ShipInfoService;
+import com.group7.ecommerce.utils.CustomUserDetails;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Locale;
+
+@RestController
+@RequestMapping("/api/customer/ship-info")
+@CrossOrigin(origins = "*")
+@RequiredArgsConstructor
+public class ShipInfoController {
+
+    private final ShipInfoService shipInfoService;
+    private final MessageSource messageSource;
+
+    /**
+     * Lấy danh sách địa chỉ giao hàng của user hiện tại
+     */
+    @GetMapping
+    public ResponseEntity<ApiResponse<List<ShipInfoResponse>>> getShipInfoList(Authentication authentication) {
+        Integer userId = getCurrentUserId(authentication);
+        List<ShipInfoResponse> shipInfos = shipInfoService.getShipInfoByUserId(userId);
```

Comment:
ri thì cái load userId mình đưa vào service làm luôn đi gái
trong method getShipInfoByUserId thì lấy userId  ra

```suggestion
        Integer userId = getCurrentUserId(authentication);
        List<ShipInfoResponse> shipInfos = shipInfoService.getShipInfoByUserId(authentication);
```